docs/templates/docker-in-workspaces.md

```diff
@@ -110,6 +111,31 @@ resource "kubernetes_pod" "dev" {
 
 > Sysbox CE (Community Edition) supports a maximum of 16 pods (workspaces) per node on Kubernetes. See the [Sysbox documentation](https://github.com/nestybox/sysbox/blob/master/docs/user-guide/install-k8s.md#limitations) for more details.
 
+## Envbox
+
+Envbox is an image developed and maintained by Coder that bundles the sysbox runtime. It works
+by starting an outer container that manages the various sysbox daemons and spawns an unprivileged
+inner container that acts as the user's workspace. The inner container is able to run system-level
+software similar to a regular virtual machine (e.g. `systemd`, `dockerd`, etc). Envbox offers the
+following benefits over running sysbox directly on the nodes:
+
+- No custom runtime installation or management on your Kubernetes nodes.
+- No limit to the number of pods that run envbox.
+
+Some drawbacks include:
+
+- The outer container must be run as privileged
+  - Note: the inner container is _not_ privileged. For more information on the security of sysbox
+    containers see sysbox's [official documentation](https://github.com/nestybox/sysbox/blob/master/docs/user-guide/security.md).
+- Initial workspace startup is slower than running `sysbox-runc` directly on the nodes. This is due
+  to `envbox` having to pull the image to its own Docker cache on its initial startup. Once the image
+  is cached in `envbox`, startup performance is similar.
```

Comment:
How does one authenticate with a private registry (e.g. ECR, Artifactory)? Could you add docs around that before we publish?

Reply:
Yup!